lib/zendesk_api/verbs.rb

```diff
@@ -44,14 +44,12 @@ def create_verb(method_verb)
           end
 
           define_method method do |*method_args|
-            begin
-              send("#{method}!", *method_args)
-            rescue ZendeskAPI::Error::RecordInvalid => e
-              @errors = e.errors
-              false
-            rescue ZendeskAPI::Error::ClientError
-              false
-            end
+            send("#{method}!", *method_args)
+          rescue ZendeskAPI::Error::RecordInvalid => e
+            @errors = e.errors
+            false
+          rescue ZendeskAPI::Error::ClientError
+            false
```

Comment:
I'm not really happy that someone looking at these lines might think I wrote this, and now just reformatted for 2.6 syntax. 🤷